spec/theme/plugins.spec.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+describe(\Dxw\MyTheme\Theme\Plugins::class, function () {
+    beforeEach(function () {
+        \WP_Mock::setUp();
+        \WP_Mock::wpFunction('esc_html', [
+            'return' => function ($a) {
+                return '_'.$a.'_';
```

Comment:
What does this give us? I see that in the test assertions we're asserting on strings with leading and trailing underscores, but I don't understand why.